src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="main">
+      <article class="calendar calendar--start-day--sun calendar--month-length--28">
```

Comment:
```suggestion
      <article class="calendar calendar__start-day--sun calendar__month-length--28">
```

Reply:
Sorry, I disagree, checklist point #2. Let's leave this to mentors

https://github.com/mate-academy/layout_calendar/blob/master/checklist.md

Reply:
Agree)